scripts/travisci/check_precommit.sh

```diff
@@ -1,15 +1,8 @@
 #!/usr/bin/env bash
 if [[ "${TRAVIS_PULL_REQUEST}" != "false" && "${TRAVIS}" == "true" ]]; then
-  files_changed=$(git diff "${TRAVIS_COMMIT_RANGE}" --name-only \
-    | grep ".*\.py")
-  if [[ ! -z "${files_changed}" ]]; then
-    pre-commit run --files  ${files_changed}
-  fi
+  pre-commit run --source ${TRAVIS_COMMIT} --origin HEAD
 else
   git remote set-branches --add origin master
   git fetch
-  files_changed=$(git diff origin/master --name-only | grep ".*\.py")
-  if [[ ! -z "${files_changed}" ]]; then
-    pre-commit run --files ${files_changed}
-  fi
+  pre-commit run --source origin/master --origin HEAD
```

Comment:
i think you might have this reversed. source == HEAD and origin == origin/master

Reply:
reading the Travis documentation, it would be more idiomatic to use TRAVIS_BRANCH instead of HEAD.

Reply:
(if so please update this case in the other CI scripts)

Reply:
I reversed it initially, but it would say no files changed. Not sure why.

Reply:
i.e.
```
autopep8 wrapper.....................................(no files to check)Skipped
Check for added large files..........................(no files to check)Skipped
Check python ast.....................................(no files to check)Skipped
Check for merge conflicts............................(no files to check)Skipped
Check Xml............................................(no files to check)Skipped
Check Yaml...........................................(no files to check)Skipped
Debug Statements (Python)............................(no files to check)Skipped
Fix End of Files.....................................(no files to check)Skipped
Forbid new submodules................................(no files to check)Skipped
Trim Trailing Whitespace.............................(no files to check)Skipped
```

Reply:
okay i think the pre-commit authors are just wrong in their naming scheme. carry on.